src/Php73/bootstrap.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Polyfill\Php73 as p;
+
+if (PHP_VERSION_ID < 70300) {
+    if (!function_exists('is_countable')) {
+        function is_countable($s) { return p\Php73::is_countable($s); }
```

Comment:
for such a simple implementation, I think we can inline the code a remove the static class (as done on other polyfills AFAIK)

Reply:
@nicolas-grekas Sure, gonna do this!